scripts/clean-rookd.sh

```diff
@@ -15,6 +15,7 @@
 # limitations under the License.
 
 sudo rm -fr /var/lib/rook
+sudo mkdir /var/lib/rook;sudo chown -R $USER:$USER /var/lib/rook
```

Comment:
yes, i was doing this as a manual step before starting rookd the next time.   much nicer to remove that manual step from my flow.